### PR TITLE
Add release script and document

### PR DESCRIPTION
**Pull request by hanahmily**
Signed-off-by: Gao Hongtao <hanahmily@gmail.com>

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -27,6 +27,7 @@ PROJECTS := banyand
 clean: TARGET=clean
 clean: PROJECTS:=$(PROJECTS) pkg
 clean: default  ## Clean artifacts in all projects
+	find . -type s -name 'localhost:*' -delete
```
Comment (lujiajing1126) [inline Makefile:30]: I recently found that `Ginkgo` provides a random port API. I will polish this behavior soon.

File: docs/release.md [hunk 2]
```diff
@@ -0,0 +1,225 @@
...
+svn co https://dist.apache.org/repos/dist/dev/skywalking/
+mkdir -p skywalking/banyandb/"$VERSION"
+cp skywalking-banyandb/build/release/skywalking-banyandb*.tgz skywalking/banyandb/"$VERSION"
+cp skywalking-banyandb/build/release/skywalking-banyandb*.tgz.asc skywalking/banyandb/"$VERSION"
+cp skywalking-banyandb/build/release/skywalking-banyandb*.tgz.sha512 skywalking/banyandb/"$VERSION"
+
+cd skywalking/banyandb && svn add "$VERSION" && svn commit -m "Draft Apache SkyWalking BanyanDB release $VERSION"
+```
+
+## Make the internal announcement
```
Comment (wu-sheng) [inline docs/release.md:41]: This is optional, do you want this? We skip this for years in many repositories.

File: docs/release.md [hunk 3]
```diff
@@ -0,0 +1,225 @@
...
+
+ * https://github.com/apache/skywalking-banyandb/tree/<Git Commit Hash>
+
+Keys to verify the Release Candidate :
+
+ * https://dist.apache.org/repos/dist/release/skywalking/KEYS
+
+Guide to build the release from source :
+
+ * https://github.com/apache/skywalking-banyandb/blob/$VERSION/docs/release.md
```
Comment (lujiajing1126) [inline docs/release.md:80]: should be `v$VERSION` here since the tag is `* (Git Tag) v$VERSION`

File: docs/release.md [hunk 4]
```diff
@@ -0,0 +1,225 @@
...
+Subject: [VOTE] Release Apache SkyWalking BanyanDB version $VERSION
+
+Content:
+
+Hi the SkyWalking Community:
+This is a call for vote to release Apache SkyWalking BanyanDB version $VERSION.
+
+Release notes:
+
+ * https://github.com/apache/skywalking-banyandb/blob/$VERSION/CHANGES.md
```
Comment (lujiajing1126) [inline docs/release.md:105]: `v$VERSION`

File: docs/release.md [hunk 5]
```diff
@@ -0,0 +1,225 @@
...
+Release Candidate:
+
+ * https://dist.apache.org/repos/dist/dev/skywalking/banyandb/$VERSION
+ * sha512 checksums
+   - sha512xxxxyyyzzz apache-skywalking-banyandb-src-x.x.x.tgz
+   - sha512xxxxyyyzzz apache-skywalking-banyandb-bin-x.x.x.tgz
+
+Release Tag :
+
+ * (Git Tag) $VERSION
```
Comment (lujiajing1126) [inline docs/release.md:116]: conflict with the description above

**Review by lujiajing1126 — CHANGES_REQUESTED** [review]

File: docs/release.md (hunk 2 above)
Comment (hanahmily) [reply docs/release.md:41]: This piece is from other sub-projects. I will remove it

File: Makefile (hunk 1 above)
Comment (hanahmily) [reply Makefile:30]: I'm suffering a massive volume of localhost socket files.

**Review by wu-sheng — APPROVED** [review]

**Review by lujiajing1126 — APPROVED** [review]
LGTM

**Pull request merged**